language/benchmarks/src/bench.move

```diff
@@ -0,0 +1,73 @@
+// `cargo bench` will call a public function in this module.
+// The Module must be called `Bench` and the set of public functions are callable from the bench (Rust code).
+// `benches/transaction.rs` contains the calling code.
+// The idea is that you build your scenario with a public entry point and a bunch of private functions as needed.
+module Bench {
+
+    //
+    // Global helpers
+    //
+    assert(check: bool, code: u64) {
+        if (check) () else abort code
+    }
+
+    //
+    // `arith` benchmark
+    //
+    public arith() {
+        let i = 0;
+        // 10000 is the number of loops to make the benchmark run for a couple of minutes, which is an eternity.
+        // Adjust according to your needs, it's just a reference
+        while (i < 10000) {
+            _ = 1;
```

Comment:
FYI, you can just do `1;`

`e;` gets is allowed as long as `e` is not a resource. And it will insert the correct number of pops